google/cloud/spanner/internal/partial_result_set_reader.h

```diff
@@ -0,0 +1,80 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef GOOGLE_CLOUD_CPP_SPANNER_GOOGLE_CLOUD_SPANNER_INTERNAL_PARTIAL_RESULT_SET_READER_H_
+#define GOOGLE_CLOUD_CPP_SPANNER_GOOGLE_CLOUD_SPANNER_INTERNAL_PARTIAL_RESULT_SET_READER_H_
+
+#include "google/cloud/spanner/result_set.h"
+#include "google/cloud/spanner/value.h"
+#include "google/cloud/optional.h"
+#include "google/cloud/status.h"
+#include "google/cloud/status_or.h"
+#include <google/spanner/v1/spanner.grpc.pb.h>
+#include <google/spanner/v1/spanner.pb.h>
+#include <grpcpp/grpcpp.h>
+#include <memory>
+
+namespace google {
+namespace cloud {
+namespace spanner {
+inline namespace SPANNER_CLIENT_NS {
+namespace internal {
+
+/**
+ * This class serves as a bridge between the gRPC `PartialResultSet` streaming
+ * reader and the spanner `ResultSet`, which is used to iterate over the rows
+ * returned from a read operation.
+ */
+class PartialResultSetReader : public internal::ResultSetSource {
+ public:
+  using GRpcReader =
```

Comment:
The count of how we spell "grpc" in google-cloud-cpp is:

```
     17 GRPC
     21 Grpc
     74 gRPC
   2151 grpc
```
It looks like `Grpc` is used most in code.

Do with this info as you will :-)